packages/studio-plugin/tests/utils/getImportSpecifier.test.ts

```diff
@@ -13,3 +16,11 @@ it("can import files in separate folders", () => {
     "../../c/d/NewFile"
   );
 });
+
+it("can preserve extensions", () => {
```

Comment:
nit: can this test and the one above be wrapped in different `describe`s indicating which function they're testing?

Reply:
done!